DevCycle.SDK.Server.Common/Model/Local/VariableCollection.cs

```diff
@@ -20,24 +20,16 @@ public bool ContainsKey(string key)
         /// Retrieve the Variable by key from the collection as the correct typeof T.
         /// Throws InvalidCastException if the requested conversion is incorrect
         /// </summary>
-        public Variable<T> Get<T>(string key)
+        public ReadOnlyVariable<object> Get(string key)
         {
-            Variable<T> existingVariable = null;
-
             if (!variables.ContainsKey(key))
                 throw new KeyNotFoundException(key);
             
             var variable = variables[key];
-            
-            if (variable != null)
-            {
-                existingVariable = variable.Convert<T>();
```

Comment:
now that you're skipping this Convert call, does all the type protection (ie. ignore server value that's the wrong type compared to default value) still work?

Reply:
yeah the `ReadOnlyVariable` in the `Variable` call gets converted to a `Variable`, which tries to cast the value to the correct type and will get caught and log a warning if they don't match

Reply:
should the convert call be removed then if it's not being used?